external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -391,19 +391,25 @@ fn parse_address_bytes(
 // Parse the beginning of an access, either an address or an identifier:
 //      LeadingNameAccess = <NumericalAddress> | <Identifier>
 fn parse_leading_name_access(context: &mut Context) -> Result<LeadingNameAccess, Box<Diagnostic>> {
-    parse_leading_name_access_(context, || "an address or an identifier")
+    parse_leading_name_access_(context, false, || "an address or an identifier")
 }
 
 // Parse the beginning of an access, either an address or an identifier with a specific description
 fn parse_leading_name_access_<'a, F: FnOnce() -> &'a str>(
     context: &mut Context,
+    global_name: bool,
```

Comment:
Maybe we should make the variant GlobalName instead of AddressName then? I think either is fine really, the name GlobalName entered my head as an alternative, and then I saw it in the flag here :)

Reply:
I will split the different and just call it a `GlobalAddress`.